src/user/dto/update-user.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { IsEmail, IsNotEmpty } from 'class-validator';
+
+export class UpdateUserDto {
```

Comment:
Can we use generic for Update and Create Dto or it's not allowed since it's the same?

Reply:
nope, we will add validators laters where the code will change